common/lib/capa/capa/templates/choicegroup.html

```diff
@@ -19,19 +19,23 @@
             <%
               label_class = 'response-label field-label label-inline'
             %>
+
             <label id="${id}-${choice_id}-label"
                 ## If the student has selected this choice...
                 % if is_radio_input(choice_id):
-                <%
-                    if status == 'correct':
+                   <%
+                    if saved_answers:
+                        correctness = status.classname
```

Comment:
Why is status just a string in the other cases below? Ideally status.classname would suffice for all.